packages/vue/src/examples/pages/product/Product.vue

```diff
@@ -472,194 +330,198 @@ export default {
-  &__description {
-    margin: var(--spacer-extra-big) 0 calc(var(--spacer-big) * 3) 0;
-    font-family: var(--font-family-secondary);
-    font-size: var(--font-size-regular);
-    line-height: 1.6;
+  &__tabs {
+    margin: var(--spacer-lg) auto var(--spacer-2xl);
     @include for-desktop {
-      font-size: var(--font-size-regular);
+      margin-top: var(--spacer-2xl);
+      --tabs-content-tab-padding: 3.5rem 0 0 0;
+    }
+    &__property {
+      margin-top: var(--spacer-lg);
+    }
+    &__review {
+      border-bottom: var(--c-light) solid 1px;
+      margin-bottom: var(--spacer-base);
+    }
+    &__pagination {
+      margin-top: 4rem;
     }
   }
-  &__properties {
-    margin: var(--spacer-big) 0 0 0;
-  }
-  &__tabs {
-    margin: var(--spacer-big) 0 0 0;
-    @include for-desktop {
-      margin: calc(5 * var(--spacer-big)) 0 0 0;
+  &__additional-info {
+    @include font(
+      --additional-info-font,
+      var(--font-light),
+      var(--font-base),
+      1.6,
+      var(--font-family-primary)
+    );
+    &__title {
+      font-weight: var(--font-bold);
+      margin: 0 0 var(--spacer-sm);
+      &:not(:first-child) {
+        margin-top: 3.5rem;
+      }
     }
-    p {
+    &__paragraph {
       margin: 0;
     }
   }
-  &__review {
-    padding: var(--spacer-big) 0;
-    border: 1px solid var(--c-light);
-    border-width: 0 0 1px 0;
-  }
-}
-.product-carousel {
-  margin: 0 calc(var(--spacer-big) * -1) 0 0;
   @include for-desktop {
-    margin: var(--spacer-big) 0;
-    --carousel-padding: var(--spacer-big);
-    --carousel-max-width: calc(100% - 13.5rem);
-  }
-}
-.product-card {
-  &:hover {
-    --product-card-box-shadow: 0 4px 20px rgba(168, 172, 176, 0.19);
+    display: flex;
   }
 }
-.images-grid {
-  max-width: 60rem;
-  margin: 0 auto;
-  &__row {
-    display: flex;
-    & + & {
-      margin: calc(var(--spacer-big) / 2) 0 0 0;
-      @include for-desktop {
-        margin: var(--spacer-big) 0 0 0;
-      }
-    }
+
+.btn {
+  &__guide,
+  &__compare,
+  &__save {
+    display: block;
+    margin: var(--spacer-xl) 0 var(--spacer-base) auto;
   }
-  &__col {
-    flex: 1;
-    margin: 0;
-    & + & {
-      margin: 0 0 0 calc(var(--spacer-big) / 2);
-      @include for-desktop {
-        margin: 0 0 0 var(--spacer-big);
-      }
-    }
+  &__compare {
+    margin-top: 0;
   }
 }
-.banner-app {
-  --banner-title-margin: var(--spacer-big) 0 0 0;
-  --banner-title-font-size: var(--h1-font-size);
-  --banner-title-font-weight: var(--h1-font-weight);
-  --banner-subtitle-font-size: var(--font-size-extra-big);
-  --banner-subtitle-font-weight: var(--body-font-weight-primary);
-  min-height: 26.25rem;
-  max-width: 65rem;
-  margin: 0 auto;
-  padding-right: calc(25% + 5rem);
-  padding-left: 2.5rem;
-  &__call-to-action {
-    display: flex;
-    margin: var(--space-big) 0 0 0;
+
+.breadcrumbs {
+  margin: var(--spacer-base) auto var(--spacer-lg);
+}
+
+@keyframes moveicon {
+  0% {
+    transform: translateY(0);
```

Comment:
@justyna-13 here is a good place to use transform3d(), thanks for that browser use GPU here :)